composer.json

```diff
@@ -22,7 +22,7 @@
     ],
     "require": {
         "php": "^7.2",
-        "friendsofsymfony/ckeditor-bundle": "^1.0",
+        "friendsofsymfony/ckeditor-bundle": "^1.0 || ^2.0",
```

Comment:
:thinking: when I merged, it looked like we just removed the bundle, but then there still is this dependency… why?

Reply:
FYI it was removed in #389 @bmaziere

Reply:
cc @core23  I think I will just remove the dependency.